Sources/SwiftClockUI/ViewModifiers/ScaleUpOnAppear.swift

```diff
@@ -7,7 +7,7 @@ struct ScaleUpOnAppear: ViewModifier {
     func body(content: Content) -> some View {
         content
             .scaleEffect(scaleValue)
-            .animation(.spring(), value: isShown)
+            .animation(isAnimationEnabled ? .spring() : nil, value: isShown)
```

Comment:
For this kind of extensions. It should be better to use explicit animation `withAnimation` like I did for `RotateOnAppear`. So maybe we could get rid of the ternary expression here.

Reply:
Done